templates/gmcq.hbs

```diff
@@ -7,6 +7,8 @@
     {{#each _items}}
     <div class="gmcq__item js-mcq-item item-{{@index}} {{odd @index}}{{#unless ../_isEnabled}}{{#if ../_canShowMarking}}{{#if _isCorrect}} is-correct{{else}} is-incorrect{{/if}}{{/if}}{{/unless}}">
 
+      <input class="gmcq__item-input js-item-input" id="{{../_id}}-{{@index}}" {{#if ../_isRadio}}name="{{../_id}}-item" {{/if}} type="{{#if ../_isRadio}}radio{{else}}checkbox{{/if}}" {{#unless ../_isEnabled}} disabled{{/unless}} aria-label="{{{a11y_normalize text}}}" data-adapt-index="{{_index}}" />
```

Comment:
please move this back down as it causes a jump on mobile if the input isn't near the icon representing it https://github.com/adaptlearning/adapt_framework/issues/2303

Reply:
Repositioned here: 

<img width="299" alt="screen" src="https://user-images.githubusercontent.com/11569678/75683232-aa454800-5c8e-11ea-9f02-e35b6e685e43.png">